frontend/src/store/modules/common/actions.ts

```diff
@@ -168,31 +168,31 @@ export default {
     async addRepo(context: any, payload: any) {
         return new Promise((resolve, reject) => {
             setTimeout(() => {
-                let rg_name = payload.rg_name || undefined
-                let repo_name = payload.repo_name || undefined
-                let repo_id = payload.repo_id || undefined
-                let repo_group_id = payload.repo_group_id || undefined
-                let gitURL = payload.url || undefined
+                const rgName = payload.rg_name || undefined
+                const repoName = payload.repo_name || undefined
+                const repoId = payload.repo_id || undefined
+                const repoGroupId = payload.repo_group_id || undefined
+                const gitURL = payload.url || undefined
 
-                let repo: Repo = context.state.AugurAPI.Repo({
-                    gitURL: gitURL,
-                    repo_id: repo_id,
-                    repo_group_id: repo_group_id,
-                    rg_name: rg_name,
-                    repo_name: repo_name
+                const repo: Repo = context.state.AugurAPI.Repo({
+                    '{gitURL}': gitURL,
+                    '{repo_id}': repoId,
+                    'repo_group_id': repoGroupId,
+                    'rg_name': rgName,
+                    '{repo_name}': repoName,
                 })
-                context.commit('mutateAPIRepo', {repo: repo, name: repo.toString()})
+                context.commit('mutateAPIRepo', { '{repo}': repo, 'name': repo.toString()})
                 resolve(repo)
             }, 2000)
         })
     },
-    async addRepoGroup(context:any, payload:any) {
-        return new Promise((resolve,reject)=>{
-            let rg_name = payload.rg_name || undefined
-            let repo_group_id = payload.repo_group_id || undefined
-            let group:RepoGroup = context.state.AugurAPI.RepoGroup({repo_group_id:repo_group_id,rg_name:rg_name})
+    async addRepoGroup(context: any, payload: any) {
+        return new Promise((resolve, reject) => {
+            const rgName = payload.rg_name || undefined
+            const repoGroupId = payload.repo_group_id || undefined
+            const group: RepoGroup = context.state.AugurAPI.RepoGroup({'{repo_group_id}': repoGroupId, '{rg_name}': rgName})
```

Comment:
Same problem mentioned above.

Reply:
The warning I was taking the advice of had mentioned that these variable types should not have any underscores (specifically, it wanted consistent and sensible casing on certain types)